app/src/main/java/com/github/onlynotesswent/ui/overview/FolderContent.kt

```diff
@@ -370,6 +335,60 @@ fun FolderContentTopBar(
             expanded = expanded,
             onFabClick = { onExpandedChange(true) },
             onDismissRequest = { onExpandedChange(false) })
+
+        // Popup for delete folder confirmation
+        if (showDeleteFolderConfirmation) {
+          ConfirmationPopup(
+              title = stringResource(R.string.delete_folder),
+              text = stringResource(R.string.confirm_delete_folder),
+              onConfirm = {
+                if (currentUser.value!!.uid == folder?.userId) {
+                  // Retrieve parent folder id to navigate to the parent folder
+                  val parentFolderId = folder.parentFolderId
+                  folderViewModel.deleteFolderById(folder.id, folder.userId)
+
+                  if (parentFolderId != null) {
+                    navigationActions.navigateTo(
+                        Screen.FOLDER_CONTENTS.replace(
+                            oldValue = "{folderId}", newValue = parentFolderId))
+                  } else {
+                    navigationActions.navigateTo(TopLevelDestinations.OVERVIEW)
+                  }
+
+                  handleSubFoldersAndNotes(
+                      folder = folder,
+                      userFolderSubFolders = userFolderSubFolders.value,
+                      userFolderNotes = userFolderNotes.value,
+                      folderViewModel = folderViewModel,
+                      noteViewModel = noteViewModel)
+                } else {
+                  Toast.makeText(
+                          context, "You are not the owner of this folder", Toast.LENGTH_SHORT)
```

Comment:
Again, we should add the toast strings to string resources

Reply:
Answered this in previous comment about toast and string resources